tests/antithesis/README.md

```diff
@@ -106,3 +108,21 @@ make antithesis-clean
 ## Troubleshooting
 
 - **Image Pull Errors**: If Docker can’t pull `etcd-client:latest`, make sure you built it locally (see the “Build and Tag” step) or push it to a registry that Compose can access.
+
+# Running Tests with Kubernetes (WIP)
+
+## Prerequisites
+
+Please make sure that you have the following tools installed on your local:
+
+- [kubectl](https://kubernetes.io/docs/tasks/tools/#kubectl)
+- [kapp](https://carvel.dev/kapp/docs/v0.64.x/install)
```

Comment:
What is this? Never heard about `kapp`. Please use standard K8s tools, not some third party.

Reply:
Usually I'd do that too (and honestly I had never used it before either).  In this case, according to antithesis's [document](https://antithesis.com/docs/getting_started/setup_k8s/#:~:text=We%20use%20kapp%20to%20deploy%20all%20your%20manifests%20with%20a%20single%20command%2C%20which%20allows%20us%20to%20avoid%20making%20assumptions%20about%20ordering%20and%20dependencies.%20It%20applies%20them%20in%20a%20logical%20order%20and%20watches%20all%20resources%20until%20they%20become%20ready.%20For%20best%20results%2C%20we%20recommend%20you%20do%20the%20same%20during%20your%20local%20testing.), they use kapp when deploying the manifests.  I think it is better to make things as close to how the deployments happen in Antithesis in local.

Reply:
Please remove